Simplenote/SPUserInterface.swift

```diff
@@ -16,7 +15,47 @@ class SPUserInterface: NSObject {
     ///
     @objc
     static var isDark: Bool {
-        // TODO: Single spot to check appearance please
-        return VSThemeManager.shared().isDarkMode()
+        if isSystemThemeSelected {
+            return isSystemInDarkMode
+        }
+
+        return Options.shared.themeName == ThemeOption.dark.themeName
+    }
+
+    /// Indicates if the System Theme is selected: Starting from +10.14, whenever the theme's name is nil
+    ///
+    @objc
+    static var isSystemThemeSelected: Bool {
+        guard #available(macOS 10.14, *) else {
+            return false
+        }
+
+        return Options.shared.themeName == nil
+    }
+
+    /// Indicates if macOS is in Dark Mode
+    ///
+    static var isSystemInDarkMode: Bool {
+        guard #available(macOS 10.14, *) else {
+            return false
+        }
+
+        return UserDefaults.standard.string(forKey: "AppleInterfaceStyle") == "Dark"
```

Comment:
Question. Why this approach and not `return NSApp.effectiveAppearance.isDark` as defined in `NSAppearance+Simplenote.swift`?  That check is already used in `NSColor+Studio.swift`. Would it make sense to use it here for consistency?

Reply:
Thank you SO MUCH for this tip!!!!